docs/api-testing/composer.md

```diff
@@ -146,34 +146,31 @@ In this method, you copy an existing body and paste it into the call.
     * Variable (the name of the variable that contains the response) - `payload`
     * Mode (the response type) - `json`
 
-      <img src={useBaseUrl('/img/api-testing/post-request-window.png')} alt="The POST request window"/>
-
-2. At the bottom of the **POST request** window, click **Add Body**.
+      
+2. Below the **POST request**, click **Add Child Component** and then click **Request Body**.
 
     <img src={useBaseUrl('/img/api-testing/post-request-add-body-nav.png')} alt="Navigating to the Post body window"/>
 
 3. Use the following in the Post body fields:
```

Comment:
The spacing/ formatting for the steps and images in both the bulleted and numbered lists is a bit inconsistent. 

For example (spacing):
<img width="774" alt="Screen Shot 2022-10-31 at 11 30 50 AM" src="https://user-images.githubusercontent.com/15352964/199050745-df12ea91-93ae-43db-b45c-771d6153658a.png">
and
<img width="711" alt="Screen Shot 2022-10-31 at 11 47 50 AM" src="https://user-images.githubusercontent.com/15352964/199050790-66e99594-e5de-4991-8658-aae9ceae06c9.png">

For example (img alignment):
<img width="806" alt="Screen Shot 2022-10-31 at 11 31 11 AM" src="https://user-images.githubusercontent.com/15352964/199051180-c5431bf9-7dec-44ab-8a46-bbbb46cd3f06.png">
and
<img width="851" alt="Screen Shot 2022-10-31 at 11 31 16 AM" src="https://user-images.githubusercontent.com/15352964/199051204-28116e66-f1b2-4658-a447-8ec48c398a67.png">

Reply:
Thanks, I will make note and fix the spacing in a separate PR.